app/lib/core/constants/bebrasBiroDropdown.dart

```diff
@@ -0,0 +1,110 @@
+class BebrasBiro {
+  String bebrasBiroUniv;
+  String valueDropdown;
+  bool isActive;
+
+  BebrasBiro({
+    required this.bebrasBiroUniv,
+    required this.valueDropdown,
+    this.isActive = true,
+  });
+
+  String userAsString() {
+    return this.bebrasBiroUniv;
+  }
+}
+
+List<BebrasBiro> bebrasBiroList = [
+  BebrasBiro(bebrasBiroUniv: "Institut Teknologi Bandung", valueDropdown: "institut_teknologi_bandung", isActive: true,),
```

Comment:
can we use the value of `valueDropdown` the same as `bebrasBiroUniv`? or just remove `valueDropdown` field
also please rename `bebrasBiroUniv` -> `bebrasBiroName`, since "Sekolah Tinggi Teknologi Garut" I think isn't a univ name ^^a
so the end result should be
```
BebrasBiro(bebrasBiroName: "Institut Teknologi Bandung", isActive: true,),
```

Reply:
I use valueDropdown to standardize values ​​with snake_case, for anticipating further use